artman/conductors/cloudtasks_conductor.py

```diff
@@ -114,14 +153,16 @@ def _prepare_dir(source_repo="https://github.com/googleapis/googleapis.git"):
     TODO(ethanbao): support loading more input files from heterogeneous data
     sources"""
 
-    repo_root = '/tmp/artman/%s' % str(uuid.uuid4())
+    task_id = str(uuid.uuid4())
```

Comment:
Thought: Would it be better to use `str(uuid.uuid4())[0:8]`? The chances of a collision are still quite low, and it is substantially easier on humans. Considering it is common in git to use only the first seven characters, even with repositories with many, many thousands of commits, I think it would be safe here.